install/helm/open-match/templates/_helpers.tpl

```diff
@@ -135,3 +155,72 @@ minReplicas: {{ .Values.global.kubernetes.horizontalPodAutoScaler.minReplicas }}
 maxReplicas: {{ .Values.global.kubernetes.horizontalPodAutoScaler.maxReplicas }}
 targetCPUUtilizationPercentage: {{ .Values.global.kubernetes.horizontalPodAutoScaler.targetCPUUtilizationPercentage }}
 {{- end -}}
+
+{{- define "openmatch.serviceAccount.name" -}}
```

Comment:
All of these names are very redundant - is there any way to simplify?

Reply:
I'm not sure what you mean here. `openmatch` is the namespace, and it's the name of the service account. Which part is looking redundant to you?

Reply:
Oh do you mean having the big-long list of named templates, rather than the name itself? If we were to drop, for example, being able to override the service names for these services, then we could just use `{{ include "openmatch.fullname" . }}-XXX` everywhere the named template is used.

We could have a name-making named template where the `XXX` above could be a parameter, if that would be more-readable at the call-site (I think it'd be aesthetically worse, personally)

However, a _big_ advantage of the named templates as they stand now, is that someone subcharting Open Match, i.e. me, doesn't need to hard-code the names of the services, I can just ask the subchart what the value of `openmatch.frontend.hostName` is in my own chart. Pretty much exactly what we do for getting the various subchart service URLs, e.g. redis, here.

Reply:
Ah, yeah that seems pretty useful.  It pains me greatly, but seems worth it.